python/spear/env.py

```diff
@@ -100,28 +100,41 @@ def _request_launch_unreal_instance(self):
             return
 
         # write temp file
-        temp_config_file = os.path.realpath(os.path.join(self._config.SPEAR.TEMP_DIR, "config.yaml"))
+        temp_dir = os.path.realpath(os.path.join(self._config.SPEAR.TEMP_DIR))
+        temp_config_file = os.path.realpath(os.path.join(temp_dir, "config.yaml"))
 
         print("[SPEAR | env.py] Writing temp config file: " + temp_config_file)
 
-        if not os.path.exists(os.path.realpath(self._config.SPEAR.TEMP_DIR)):
-            os.makedirs(os.path.realpath(self._config.SPEAR.TEMP_DIR))
+        if not os.path.exists(temp_dir):
+            os.makedirs(temp_dir)
         with open(temp_config_file, "w") as output:
             self._config.dump(stream=output, default_flow_style=False)
 
         # create a symlink to SPEAR.DATA_DIR
         if self._config.SPEAR.DATA_DIR != "":
 
-            paks_dir = os.path.join(self._config.SPEAR.CONTENT_DIR, "Paks")
+            assert os.path.exists(self._config.SPEAR.DATA_DIR)
+
+            if sys.platform == "win32":
+                paks_dir = os.path.realpath(os.path.join(os.path.dirname(os.path.realpath(args.executable)), "..", "..", "Content", "Paks"))
```

Comment:
`args.executable` is not valid here. Similarly, `args.data_dir`.

Reply:
I think we should change it to `self._config.SPEAR.STANDALONE_EXECUTABLE` and add another condition in the if statement of #114,
```python 
if self._config.SPEAR.STANDALONE_EXECUTABLE != "" and self._config.SPEAR.DATA_DIR != "":
```

Reply:
If we need to support symlink creation for both editor mode and standalone mode, then we will have to handle these paths differently.

Reply:
Good catch. I think if you're running in editor mode for some reason, then you don't need pak files. So we only need to do this step in standalone mode. I updated the logic accordingly.

Reply:
I changed it to:

```
        if self._config.SPEAR.LAUNCH_MODE == "standalone_executable" and self._config.SPEAR.DATA_DIR != "":

            assert os.path.exists(self._config.SPEAR.STANDALONE_EXECUTABLE)
            assert os.path.exists(self._config.SPEAR.DATA_DIR)
```